js/base/main.js

```diff
@@ -1,6 +1,7 @@
 import "ripe-sdk/src/css/ripe.css";
 
 import { Ripe } from "ripe-sdk";
+import { API as RipeIdAPI } from "ripe-id-api";

```

Comment:
@BeeMargarida let's keep this scoped to where it's being used: the technical plugin. Make this PR only about adding the button to commons.

Reply:
@gcandal So I only initialize this in the ripe id plugin?

Reply:
yes :)